batect.yml

```diff
@@ -94,4 +94,5 @@ tasks:
       - buildFunTest
     run:
       container: run
-      command: ./gradlew funTest
+      command: "'./gradlew funTest jacocoFunTestReport -p analyzer'"
```

Comment:
Can you please explain what this does / why it is necessary? The commit message unfortunately does not give me any clue. In particular:

- Why are JaCoCo reports being created now?
- Why is running the tests now limited to functional tests in the analyzer?

Reply:
@heliocastro removed `jacocoFunTestReport -p analyzer` but now there is below error.

@sschuberth  Any clue what is causing this?

```

org.ossreviewtoolkit.model.utils.PostgresFileArchiverStorageFunTest > Before Spec Error FAILED
    java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
        at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$4(DockerClientProviderStrategy.java:156)
        at java.base/java.util.Optional.orElseThrow(Optional.java:408)
        at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:148)
        at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:146)
        at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:188)
        at org.testcontainers.DockerClientFactory$1.getDockerClient(DockerClientFactory.java:101)
        at com.github.dockerjava.api.DockerClientDelegate.authConfig(DockerClientDelegate.java:107)
        at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:316)
        at com.opentable.db.postgres.embedded.EmbeddedPostgres.<init>(EmbeddedPostgres.java:99)
        at com.opentable.db.postgres.embedded.EmbeddedPostgres$Builder.start(EmbeddedPostgres.java:353)
        at org.ossreviewtoolkit.utils.test.PostgresListener.beforeSpec(PostgresListener.kt:42)
        at io.kotest.engine.spec.SpecExtensions.beforeSpec-gIAlu-s(SpecExtensions.kt:49)
        at io.kotest.engine
```

Reply:
I don't have a concrete idea, but it looks like some of the changes broke running Docker-in-Docker, as our `PostgresFileArchiverStorageFunTest` relies on `org.testcontainers`.